app/src/main/java/com/telnyx/webrtc/sdk/ui/MainActivity.kt

```diff
@@ -95,6 +95,17 @@ class MainActivity : AppCompatActivity() {
             dialog.show()
             true
         }
+        R.id.action_wsmessages -> {
+            if (wsMessageList == null) {
+                wsMessageList = ArrayList()
+            }
+            val instanceFragment = WsMessageFragment.newInstance(wsMessageList)
+            supportFragmentManager.beginTransaction()
+                .replace(android.R.id.content, instanceFragment)
+                .addToBackStack(null)
+                .commitAllowingStateLoss()
```

Comment:
Is there a reason we're using this instead of .commit()?

Reply:
as the fragment transaction is not very critical for wsmessages I opted for commit with state loss.